src/sinks/gcp/stackdriver_logs.rs

```diff
@@ -36,6 +36,9 @@ enum HealthcheckError {
 #[derive(Deserialize, Serialize, Debug, Clone, Default)]
 #[serde(deny_unknown_fields)]
 pub struct StackdriverConfig {
+    #[serde(skip, default = "default_endpoint")]
+    endpoint: String,
+
```

Comment:
I don't see a docs change to add this config item, which should also be noted in the announcement.

Reply:
It's marked `skip` and always has a default, so it's not part of the user-facing configuration.